### DISCUSSION TITLE
Is it possible to set NVD mirroring frequency to every 6 hours (4 times per day)

### DISCUSSION THREAD

**Discussion by Souhila99** [DependencyTrack/dependency-track#2612] [Q&A] [answered]
Our enterprise is considering setting the NVD mirroring frequency from every 24 hours to every 6 hours. But according to the documentation, it's not recommended to choose short intervals : https://docs.dependencytrack.org/getting-started/recurring-tasks/#configuration. 
Why is choosing short intervals not recommended and what are the possible issues we may encounter ?

**Comment by stevespringett** [comment, marked as answer]
Sure, it's possible, but there is no benefit of doing so. Refer to https://nvd.nist.gov/vuln/data-feeds which states:
> Each feed is updated nightly, but only if the content of that feed has changed.

For this reason, the DT documentation recommends mirroring every 24 hours to align with the update frequency of the NVD.

**Reply by Souhila99** [reply]
Hi @stevespringett, thanks a lot for you response. I have one more question please, do you have an approximate date when dependency track will migrate from data feeds to NVD APIv2 ? and in this case, will you keep the same frequency ?

**Comment by stevespringett** [comment]
> do you have an approximate date when dependency track will migrate from data feeds to NVD APIv2 

No, I do not, but soon.

> and in this case, will you keep the same frequency 

More frequent - every 6 is likely good for most cases
